Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -88,3 +88,55 @@ private extension BiometricAuthentication.Biometry {
         }
     }
 }
+
+// MARK: - Account Deletion
+extension SPSettingsViewController {
+
+    @objc
+    func deleteAccount() {
```

Comment:
Suggestion:

A. `func deleteAccountWasPressed()`
B. `func presentAccountDeletionConfirmation`

Why: because `deleteAccount` would imply that the account is immediately deleted, but there's a confirmation step beforehand